components/table/test/table.sticky.visual-diff.html

```diff
@@ -229,6 +356,23 @@
 				</table>
 			</d2l-table-wrapper>
 		</div>
+		<div class="visual-diff">
+			<d2l-table-wrapper sticky-headers id="one-column">
```

Comment:
This one is actually broken (no bottom border), so I'll try to fix it with the new code.